.rubocop.yml

```diff
@@ -1,3 +1,7 @@
+inherit_mode:
+  merge:
+    - Exclude
```

Comment:
Added this because otherwise rubocop picks up rules from some gem in our cache, background in https://github.com/ruby/setup-ruby/issues/136

You can see example of the failure at https://github.com/dentarg/influxdb-ruby/runs/1825374738?check_suite_focus=true#step:5:9